src/message.rs

```diff
@@ -480,11 +481,43 @@ pub struct Message {
     pub(crate) param: Params,
 }
 
+impl Default for Message {
+    fn default() -> Self {
+        Self {
+            id: Default::default(),
+            from_id: Default::default(),
+            to_id: Default::default(),
+            chat_id: Default::default(),
+            viewtype: Default::default(),
+            state: Default::default(),
+            download_state: Default::default(),
+            hidden: Default::default(),
+            timestamp_sort: Default::default(),
+            timestamp_sent: Default::default(),
+            timestamp_rcvd: Default::default(),
+            ephemeral_timer: Default::default(),
+            ephemeral_timestamp: Default::default(),
+            text: Default::default(),
+            subject: Default::default(),
+            rfc724_mid: create_outgoing_rfc724_mid(),
+            in_reply_to: Default::default(),
```

Comment:
Not using default values kind of destroys the purpose of default, so I'm also fine with changing the uses of default as it only has 4 occurences.

Reply:
Yes, I think it's a bit surprising to have a default implementation that isn't using default values, so it would be nicer to change it